src/postamble_minimal.js

```diff
@@ -233,9 +242,7 @@ WebAssembly.instantiate(Module['wasm'], imports).then(function(output) {
   }
 #endif
 #endif // WASM == 2
-})
 #endif // ASSERTIONS || WASM == 2
-;
+});
```

Comment:
a little hard to follow these `});` scopes, but as we do test minimal runtime with assertions (e.g. in wasm0) should be good.